examples/angular/package.json

```diff
@@ -42,6 +42,7 @@
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "~1.5.0",
     "protractor": "~7.0.0",
+    "serve": "^12.0.0",
```

Comment:
http-server also works, but I found `serve` to be a bit easier for SPA routed apps. http-server needed a few more flags than above to work.